src/System.Memory/src/System/Buffers/ReadOnlySequence_helpers.cs

```diff
@@ -277,113 +277,57 @@ private void BoundsCheck(in SequencePosition start, in SequencePosition end)
 
         internal bool TryGetReadOnlySequenceSegment(out ReadOnlySequenceSegment<T> startSegment, out int startIndex, out ReadOnlySequenceSegment<T> endSegment, out int endIndex)
         {
-            GetTypeAndIndices(Start.GetInteger(), End.GetInteger(), out SequenceType type, out startIndex, out endIndex);
```

Comment:
Get rid of GetTypeAndIndices method. After https://github.com/dotnet/corefx/pull/29135, no one is calling it anymore.

Reply:
I do not use GetTypeAndIndices

Reply:
Yes I know, we can remove that helper method. No one uses it.

Reply:
@AlexRadch, can you please remove the definition of the GetTypeAndIndices() method, and then we can merge this in. Thanks.